GetDisplayUserInfo-McGuire/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GetDisplayUserInfo_McGuire
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hey there! I just need some basic info!");
+
+            string firstName = GetStringUserInfo("What is your first name?");
+            string lastName = GetStringUserInfo("What is your last name?");
+            int favoriteNumber = GetIntUserInfo("What is your favorite number?");
+            int numberOfPets = GetIntUserInfo("How many pets do you have?");
+            List<string> nameOfPets = PetNames("What are the names of your pets?", numberOfPets);
+            
+
+            Console.WriteLine("Your name is " + firstName + " " + lastName + ".");
+            Console.WriteLine("Your favorite number is " + favoriteNumber + ".");
+            Console.WriteLine("You have " + numberOfPets + " pets.");
+            Console.WriteLine("Your pet names are: ");
+            foreach (var name in nameOfPets)
```

Comment:
This works, but I usually recommend avoiding `var` while still learning. It's good and considered by many to be best practices, but this line could be written explicitly as `foreach (string name in nameOfPets)`